backend/entities/views.py

```diff
@@ -70,7 +75,7 @@ def destroy(self, request: Request, *args: str, **kwargs: int) -> Response:
         instance = get_object_or_404(Organization, pk=kwargs["pk"])
         instance.delete()
         data = {"message": f'Organization {kwargs["pk"]} has been deleted successfully'}
-        return Response(data, status=status.HTTP_204_NO_CONTENT)
+        return Response(status=status.HTTP_202_ACCEPTED)
```

Comment:
Minor thing here. Accepted is good status for this but we generally use the `status.HTTP_201_CREATED` in case we create an instance.